biojava-structure/src/test/java/org/biojava/nbio/structure/TestAtomCache.java

```diff
@@ -227,7 +228,7 @@ public void testGetScopDomain() throws IOException, StructureException {
 
 		Structure s = cache.getStructure(name);
 		assertNotNull("Failed to fetch structure from SCOP ID", s);
-		assertEquals("2gs2.A", s.getName());
+		assertEquals("2GS2.A", s.getName()); //TODO make sure that updating the case won't cause any harm
```

Comment:
Fetching lowercase structures still works, right? Maybe we should leave some tests lowercase to check this.

Reply:
Fetching the lowercase structures still works, because it is converted to lower case anyway in the URL.